doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -149,6 +149,8 @@ nRF5340 Audio
 
   * LE Audio Controller Subsystem for nRF53 from version 3303 to version 3307.
   * Removed support for the nRF5340 Audio DK (PCA10121) board version 0.7.1 or older
+  * Power module has been re-factored so that it uses upstream Zephyr INA23X
+    sensor driver.
```

Comment:
```suggestion
  * Power module has been re-factored so that it uses upstream Zephyr INA23X sensor driver.
```
We follow "one sentence per line" rule for the NCS docs.

Reply:
fixed